src/System.Security.Cryptography.Algorithms/ref/System.Security.Cryptography.Algorithms.cs

```diff
@@ -11,13 +11,14 @@ namespace System.Security.Cryptography
     public abstract partial class Aes : System.Security.Cryptography.SymmetricAlgorithm
     {
         protected Aes() { }
-        public static System.Security.Cryptography.Aes Create() { throw null; }
-        public static System.Security.Cryptography.Aes Create(string algorithmName) { throw null; }
+        public static new System.Security.Cryptography.Aes Create() { throw null; }
```

Comment:
Note I also ran into this build issue locally (before seeing this PR) and just fixed up the Algorithms contracts and merged it (to add the new modifier to all such static Create methods)
https://github.com/dotnet/corefx/commit/ad0c8cd828c5975991107487a5fed8f578892a2a